src/main/java/com/library/model/entity/postgres/Author.java

```diff
@@ -31,10 +31,10 @@ public class Author extends ParentEntity {
     private String surname;
 
     @NotNull
-    @Column
+    @Column(name = "birth_date")
```

Comment:
Custom name definition is unnecessary in @Column

Reply:
I got exception without "name"

Reply:
Need some details about exception.
And why it worked fine before?

Reply:
Already resolved this point: add to hibernate properties implicit_naming_strategy and physical_naming_strategy